srcpkgs/libadwaita/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'libadwaita'
 pkgname=libadwaita
-version=1.0.2
+version=1.1.0
```

Comment:
please split this update to a separate PR, you can leave the comment here until the PR is merged and remove it then